bpf/lib/ipv4.h

```diff
@@ -8,6 +8,30 @@
 
 #include "dbg.h"
 
+struct ipv4_frag_id {
+	__be32	daddr;
+	__be32	saddr;
+	__be16	id;		/* L4 datagram identifier */
+	__u8	proto;
+	__u8	pad;
+} __attribute__((packed));
+
+struct ipv4_frag_l4ports {
+	__be16	sport;
+	__be16	dport;
+} __attribute__((packed));
+
+#if defined IPV4_FRAGMENTS
+struct bpf_elf_map __section_maps IPV4_FRAG_DATAGRAMS_MAP = {
+	.type           = BPF_MAP_TYPE_LRU_HASH,
+	.size_key	= sizeof(struct ipv4_frag_id),
+	.size_value	= sizeof(struct ipv4_frag_l4ports),
+	.pinning	= PIN_GLOBAL_NS,
+	.max_elem	= CILIUM_IPV4_FRAG_MAP_MAX_ENTRIES,
+	.flags		= CONDITIONAL_PREALLOC,
```

Comment:
Does LRU hash even work with no prealloc?

Reply:
kernel/bpf/hashtab.c says:
``` c
	if (lru && !prealloc)
		return -ENOTSUPP;
```
So, no. Uh, I probably copied this flag and forgot to check its meaning :fearful:. Let's remove it quick \<insert parrot>!